style3.css

```diff
@@ -116,13 +97,54 @@ ul li:hover #a{
 
 
 
+/* Set up the footer */
 /* Set up the footer */
 .footer{
 	background-color: #0C59CF;
+	grid-column-start: 1;
 	grid-column-end: span 4;
-	padding-top:25px;
 	text-align: center;
-	color:black;
 	height:50px;
 	color:white;
+	padding-top:20px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+
+
+user1{
```

Comment:
27/09/2021: I create the the user1 which hold the setting for the heading of the User column. Because it was heading so I decided to apply the heading font and font-color. Also I set position for it so it can display in a nice place.